### DISCUSSION TITLE
How to put loading emoji/gif into a message

### DISCUSSION THREAD

**Discussion by johnlim5847** [discordjs/discord.js#9162] [Q&A] [answered]
https://user-images.githubusercontent.com/3675946/220817924-264b1d04-a167-434a-9ca8-21df5b6ee1b5.mp4

Hey guys, Just wondering is there any default function that can invoke the loading emoji/gif when bot reply a message like the video above. If no, how can I achieve this outcome? Thanks!

**Comment by almostSouji** [comment, marked as answer]
This is a deferred response and we explain the procedure in the guide here: <https://discordjs.guide/slash-commands/response-methods.html#deferred-responses>. This serves as an initial response, which you need to send within 3 seconds. You can then respond to it within 15 minutes (then the interaction token expires)

**Reply by johnlim5847** [reply]
Thanks for your answer @almostSouji !
